### ISSUE TITLE
Field type text required

### ISSUE THREAD

**Issue by viniciusjau** [omarshammas/jquery.formance#21]
Hi guys!

How create a field type text required?
To use in a field with the name...

**Comment by jh3y** [comment]
Hey @viniciusjau,

Could you elaborate?

If you have a field that is of type text and required, you could simply use the HTML5 attribute `required` attribute if using a browser > IE9 and !Safari. It also depends on the context and set up of your project. For example, something like Angular has this functionality available within its basic validation directives. Or jQuery validate also offers a solution to requiring input in a field.

Formance caters to more specific fields with regards to their validation and formatting.

@jh3y

**Comment by viniciusjau** [comment]
Yes could use the HTML5 attribute.. but I would not get this effect
![form](https://cloud.githubusercontent.com/assets/11538313/6717447/3285487a-cd8b-11e4-9b43-ba7800d57ac9.jpg)

**Comment by jh3y** [comment]
The effect is achieved using CSS. You could write some CSS and apply it to your input. For the `invalid` label either opt to use maybe a CSS pseudo element or a styled label that appears in a show/hide manner.

@jh3y

**Comment by autoboxer** [comment]
I agree, having a base required field type would be very helpful.  The benefit of a library like this is consistency through the validation handling.  Most users will have a need for this basic functionality and won't want to roll their own solution using the HTML5 attribute.
